airsonic-main/src/main/java/org/airsonic/player/service/search/FieldNames.java

```diff
@@ -0,0 +1,74 @@
+package org.airsonic.player.service.search;
+
+/**
+ * Enum that symbolizes the field name used for lucene index.
+ * This class is a division of what was once part of SearchService and added functionality.
+ */
+class FieldNames {

```

Comment:
Any particular reason for the changing of the field names?

Reply:
The value has been changed simply to shorten the query.
(I just want to reduce the line breaks in assert code in [QueryFactoryTestCase](https://github.com/airsonic/airsonic/blob/53698e0d1f445edde555f67cecc166194710ff69/airsonic-main/src/test/java/org/airsonic/player/service/search/QueryFactoryTestCase.java)
Conversely, if you are reducing the readability of the query, restore it.


The reason for making a literal into another class is that it is not originally a value that SearchService should hold by design.
The class name has been FieldNames to avoid confusion with the org.apache.lucene.document.Field class

Reply:
I revised that according to your advice.